apps/src/p5lab/P5Lab.js

```diff
@@ -213,10 +213,11 @@ P5Lab.prototype.init = function(config) {
   this.skin = config.skin;
   if (this.isSpritelab) {
     const MEDIA_URL = '/blockly/media/spritelab/';
-    this.skin.smallStaticAvatar = MEDIA_URL + 'avatar.png';
-    this.skin.staticAvatar = MEDIA_URL + 'avatar.png';
-    this.skin.winAvatar = MEDIA_URL + 'avatar.png';
-    this.skin.failureAvatar = MEDIA_URL + 'avatar.png';
+    const avatar = config.level.instructionsIcon || 'avatar';
+    this.skin.smallStaticAvatar = `${MEDIA_URL}${avatar}.png`;
+    this.skin.staticAvatar = `${MEDIA_URL}${avatar}.png`;
+    this.skin.winAvatar = `${MEDIA_URL}${avatar}.png`;
+    this.skin.failureAvatar = `${MEDIA_URL}${avatar}.png`;
```

Comment:
nit: i think this would be more readable if we set `${MEDIA_URL}${avatar}.png` to a variable and reused that. something like

```javascript
const avatarUrl = `${MEDIA_URL}${config.level.instructionsIcon || 'avatar'}.png`
this.skin.smallStaticAvatar = avatarUrl;
// set other avatar skins
```